docs/installationguide/ubuntu.rst

```diff
@@ -198,7 +198,7 @@ As of Oct 3 2015, zm.conf is not readable by ZM. This is likely a bug and will g
 
 If you changed the  DB password **after** installing ZM, the APIs will not be able to connect to the DB.
 
-If you have, go to zoneminder/www/api/app/Config & Edit ``database.php``
+If you have, go to ``/usr/share/zoneminder/www/api/app/Config`` & Edit ``database.php``
```

Comment:
Not sure it makes sense to put an absolute path here, as ZM can be installed to many locations.  That said, this document does assume you're using the PPA, which installs there.  So nevermind.